app/server/lib/AttachmentStoreProvider.ts

```diff
@@ -95,3 +121,45 @@ export async function checkAvailabilityAttachmentStoreOptions(options: ICreateAt
     unavailable: options.filter((option, index) => !availability[index]),
   };
 }
+
+// Make a filesystem store that will be cleaned up on process exit.
+// Can be replaced by the full config in the future - this is temporary until then.
```

Comment:
Convention in this repo is to add a line in the form:
```
// TODO: ...
```
when making promises like this in the code (so we can easily find those promises).

Reply:
If only relevant for tests, please say so.